"These apps are now globally available" is confusing now that --global is a thing

```
$ pipx --version
1.6.0
$ pipx install cowsay
  installed package cowsay 6.1, installed using Python 3.11.9
  These apps are now globally available
    - cowsay
done! ✨ 🌟 ✨
```

The phrasing "These apps are now globally available" seems to imply that I asked for a system-wide install using `--global`, but I didn't (and it didn't). This is confusing.

Alternatives include:

- Just writing "These apps are now available"
- Writing "These apps are now available user-wide" or "These apps are now available system-wide" depending on whether `--global` was passed

Hi everyone,
I saw the tag "good first issue" and tried to help solve it.
I edited the file `commands/common.py` on line 352 with the first alternative suggested but while running the test I encountered some failures:

```
conda create --name pipx-test python pip
conda activate pipx-test
python -m pip install -e .
python -m pip install --user nox
nox -s tests-3.12
```

> ================================ short test summary info ================================
FAILED tests/test_environment.py::test_cli - AssertionError: assert False
FAILED tests/test_install.py::test_install_fetch_missing_python_invalid[3.1] - AssertionError: assert 0
= 2 failed, 344 passed, 8 skipped, 124 deselected, 1 xpassed, 7 warnings in 879.96s (0:14:39) =
nox > Command pytest --cov=pipx --cov-report= tests failed with exit code 1
nox > Session tests-3.12 failed.

As I don't think the line in itself is wrong (I mean, I only changed a string), I may need some help understanding the tests work so  that I can contribute correctly. Any suggestion is welcome.

As we are on the topic, @dechamps, do you think it would be better to also change line 358 from `These manual pages are now globally available` to `These manual pages are now available`?

--- a/src/pipx/commands/common.py
+++ b/src/pipx/commands/common.py
@@ -352,13 +352,13 @@ def _get_list_output(
     )
 
     if new_install and (exposed_binary_names or unavailable_binary_names):
-        output.append("  These apps are now globally available")
+        output.append("  These apps are now available")
     output.extend(f"    - {name}" for name in exposed_binary_names)
     output.extend(
         f"    - {red(name)} (symlink missing or pointing to unexpected location)" for name in unavailable_binary_names
     )
     if new_install and (exposed_man_pages or unavailable_man_pages):
-        output.append("  These manual pages are now globally available")
+        output.append("  These manual pages are now available")
     output.extend(f"    - {name}" for name in exposed_man_pages)
     output.extend(
         f"    - {red(name)} (symlink missing or pointing to unexpected location)" for name in unavailable_man_pages

--- a/tests/test_install_all_packages.py
+++ b/tests/test_install_all_packages.py
@@ -275,7 +275,7 @@ def verify_installed_resources(
         return True
 
     reported_resources_re = re.search(
-        r"These " + resource_name_long + r" are now globally available\n((?:    - [^\n]+\n)*)",
+        r"These " + resource_name_long + r" are now available\n((?:    - [^\n]+\n)*)",
         captured_outerr.out,
         re.DOTALL,
     )

